### PR TITLE
Curve fill: draw line around patch 

### PR DESCRIPTION
**Pull request by SamSchott**
This is a matter of personal taste, but it sometimes looks a lot more refined when drawing a line around the fill-patch. For instance this example, created with pyqtgraph:

mercurygui screenshot

### CODE REVIEW

**Comment by 2xB** [comment]
Would this prevent you from different fill and line colors? Then I would suggest to make it optional.

Btw. if you can, it would be great if you created patches fresh from the develop branch, because then only the commits really associated with the pull request show up.

**Comment by SamSchott** [comment]
> Would this prevent you from different fill and line colors? Then I would suggest to make it optional.

No, it wouldn't. Fill and line colors will still be independent, but the line will be drawn all around the fill in addition to the plot itself. This is really a personal preference, so it may be a good idea it make it optional anyways.

> Btw. if you can, it would be great if you created patches fresh from the develop branch, because then only the commits really associated with the pull request show up.

Of course, sorry about that.

File: pyqtgraph/graphicsItems/PlotCurveItem.py [hunk 1]
```diff
@@ -502,7 +502,11 @@ def paint(self, p, opt, widget):
             p.setPen(sp)
             p.drawPath(path)
         p.setPen(cp)
-        p.drawPath(path)
+        if self.fillPath is not None:
+            p.drawPath(self.fillPath)
+        else:
+            p.drawPath(path)
+        profiler('drawPath')
```
Comment (j9ac9k) [inline pyqtgraph/graphicsItems/PlotCurveItem.py]: Mind removing the duplicate call to `profiler('drawPath')`?

Comment (SamSchott) [reply pyqtgraph/graphicsItems/PlotCurveItem.py]: Done.

**Comment by j9ac9k** [comment]
Thanks for the PR @SamSchott LGTM, merging

**Comment by 2xB** [comment]
Since I saw this now "in the real world": For scientific visualizations a plotted line has a very precise meaning and multiple lines in the same color can e.g. mean a bifurcation. Therefore, I'd like to go with @SamSchott s suggestion of making it optional.

**Pull request merged**